packages/amplify-util-uibuilder/package.json

```
@@ -24,6 +24,7 @@
     "tiny-async-pool": "^2.1.0"
   },
   "devDependencies": {
+    "@aws-amplify/appsync-modelgen-plugin": "^2.3.3",
```

Just a nitpick since yarn resolves this dependency to the same version, but can this match the version codegen depends on:
https://github.com/aws-amplify/amplify-cli/pull/12228/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR144

```suggestion
    "@aws-amplify/appsync-modelgen-plugin": "^2.4.0",
```

---

Made the change. Just curious, where did you get the version? I am looking at the `package.json` in `amplify-codegen`, but I guess I'm looking at the wrong place. https://github.com/aws-amplify/amplify-codegen/blob/main/package.json

---

There's probably a simpler way to do it, but I searched inside our yarn.lock file: https://github.com/aws-amplify/amplify-cli/blob/dev/yarn.lock#L9092